domain.go

```diff
@@ -26,29 +26,41 @@ type DomainInput struct {
 	Note        *string `json:"note,omitempty"`
 }
 
-func (s *DomainId) ChildSystems(client *Client, variables *PayloadVariables) (*SystemConnection, error) {
+func (d *Domain) GetTags(client *Client, variables *PayloadVariables) (*TagConnection, error) {
+	return d.DomainId.Tags(client, variables)
```

Comment:
We should put this on the lower composite object

```suggestion
func (d *DomainId) GetTags(client *Client, variables *PayloadVariables) (*TagConnection, error) {
	return d.Tags(client, variables)
```